src/libraries/System.IO.Pipes/tests/PipeStreamConformanceTests.cs

```diff
@@ -804,15 +804,12 @@ public sealed class NamedPipeTest_ServerIn_ClientOut_Asynchronous : NamedPipeTes
         protected override PipeOptions Options => PipeOptions.Asynchronous;
     }
 
-    public sealed class NamedPipeTest_ServerInOut_ClientInOut_Synchronous : NamedPipeTest_ServerInOut_ClientInOut
-    {
-        protected override PipeOptions Options => PipeOptions.None;
-
-        // TODO https://github.com/dotnet/runtime/issues/72526:
-        // The ConcurrentBidirectionalReadsWrites_Success test hangs on Windows with PipeOptions.None and InOut named pipes.
-        // Disabling for now.
-        protected override bool SupportsConcurrentBidirectionalUse => !OperatingSystem.IsWindows();
-    }
+    // [ActiveIssue("https://github.com/dotnet/runtime/issues/72526")]
+    // Multiple tests hang in this configuration.
+    //public sealed class NamedPipeTest_ServerInOut_ClientInOut_Synchronous : NamedPipeTest_ServerInOut_ClientInOut
+    //{
+    //    protected override PipeOptions Options => PipeOptions.None;
+    //}
```

Comment:
`ActiveIssueAttribute` contains `AttributeTargets.Class`, I assume you commented it all because it is not really disabling anything?

Reply:
Yeah, I just did it so that later searches might find it.